integration_tests/conftest.py

```diff
@@ -21,8 +21,17 @@ def pytest_addoption(parser):
         "--no-cleanup",
         action="store_true",
         default=False,
-        help="Keep the coverage report in `kcov_output` directory. If this flag is not provided, "
-             "both coverage related directories are removed."
+        help="Keep the coverage report in `kcov_output` directory. If this "
+             "flag is not provided, both coverage related directories are "
+             "removed."
+    )
+    parser.addoption(
```

Comment:
I find it a bit strange to have this configuration here because this parameter is actually valid just for one test. Of course, same goes for `--no-cleanup`. Can we at least add a TODO to find a better way to provide parameters to tests?

Reply:
I don't like the current approach either, but couldn't find an alternative that keeps using pytest. Added a TODO to look for another way.